deploy-edgecluster/30-nodenet.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+Environment="KUBELET_NODE_IP=$NODE_IP KUBELET_NODE_IPS=$NODE_IP"
```

Comment:
It failed to start with the above line.
Was able to start with the following (note the quotations):
Environment="KUBELET_NODE_IP=192.168.7.10" "KUBELET_NODE_IPS=192.168.7.10"